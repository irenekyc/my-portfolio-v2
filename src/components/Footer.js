import React from 'react'

const Footer =()=>{
    return (
        <footer>
        <p> Design and built by Irene K</p>
    </footer>
    )
}

export default Footer