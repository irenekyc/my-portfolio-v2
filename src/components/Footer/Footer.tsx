import React from "react";

const Footer = () => {
  return (
    <footer data-testid="footer">
      <p> Design and built by Irene K</p>
    </footer>
  );
};

export default Footer;
