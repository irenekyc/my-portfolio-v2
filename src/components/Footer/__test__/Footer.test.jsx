import React from "react";
import Footer from "../index";
import { render, screen } from "@testing-library/react";

it("Render Footer Section", () => {
  render(<Footer />);
  const footer = screen.getByTestId("footer");
  expect(footer).toBeInTheDocument();
  expect(footer).toMatchSnapshot();
});
