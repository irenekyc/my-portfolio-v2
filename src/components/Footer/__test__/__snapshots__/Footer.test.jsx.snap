// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer Section 1`] = `
<footer
  data-testid="footer"
>
  <p>
     Design and built by Irene K
  </p>
</footer>
`;
